metricbeat/module/docker/helper.go

```diff
@@ -35,37 +51,25 @@ func ExtractContainerName(names []string) string {
 	}
 	return strings.Trim(output, "/")
 }
+
 func BuildLabelArray(labels map[string]string) []common.MapStr {
 
-	output_labels := make([]common.MapStr, len(labels))
+	outputLabels := make([]common.MapStr, len(labels))
 	i := 0
 	var keys []string
 	for key := range labels {
 		keys = append(keys, key)
 	}
 	sort.Strings(keys)
 	for _, k := range keys {
+		// Replace all . in the labels by _
+		// TODO: WHY?
```

Comment:
I was first thinking . were replaced by _ because in ES <2.4 . can't be used in field names. But it seems to be the names where it is replaced? Perhaps there are some disadvantages in Kibana with the . ? Would be interesting to get some more background info on this decision.